inc/Engine/Optimization/Minify/CSS/AbstractCSSOptimization.php

```diff
@@ -121,11 +121,7 @@ protected function is_minify_excluded_file( array $tag ) {
 			return true;
 		}
 
-		if ( false !== strpos( $tag[0], 'media=' ) && ! preg_match( '/media=["\'](?:\s*|[^"\']*?\b(all|screen)\b[^"\']*?)["\']/i', $tag[0] ) ) {
-			return true;
-		}
-
-		if ( false !== strpos( $tag[0], 'only screen and' ) ) {
+		if ( false !== strpos( $tag[0], 'media=' ) && ! preg_match( '/media=["\'](?:\s*|[^"\']*?\b(?:\s*?,\s*?)?(all|screen)(?:\s*?,\s*?)?)["\']/i', $tag[0] ) ) {
```

Comment:
```suggestion
		if ( false !== strpos( $tag[0], 'media=' ) && ! preg_match( '/media=["\'](?:\s*|[^"\']*?\b(all|screen)\b[^"\']*?)["\']/i', $tag[0] ) ) {
```

Reply:
So we removed any check on media attribute for minify only and kept this check only for combining (local and external files)